content/en/cosign/openid_signing.md

```diff
@@ -4,71 +4,40 @@ category: "Cosign"
 position: 115
 ---
 
----------
-
-**NOTE:** This page contains outdated information. Identity-based signing is now fully supported in Cosign.
-
----------
-
-The full design document for this can be found [here](https://docs.google.com/document/d/189w4Fp1GEA1b2P633HyqTwtcWFNTu_Af4meolMa_1_8/edit?resourcekey=0-QoqNqcHXvSuPnMUdn8RGOQ#heading=h.2mtrw7byet02)
-(join sigstore-dev@googlegroups.com for access).
-
-This document explains how the experimental keyless signatures work in Cosign.
+This document explains how keyless signatures work in Cosign.
 
 To learn more about OIDC, please review [OIDC Usage in Fulcio](/fulcio/oidc-in-fulcio/).
 
-## Usage
-
-Keyless signing:
-
-```shell
-$ COSIGN_EXPERIMENTAL=1 cosign sign gcr.io/user-vmtest2/demo
-Generating ephemeral keys...
-Retrieving signed certificate...
-Your browser will now be opened to:
-https://accounts.google.com/o/oauth2/v2/auth?access_type=offline&client_id=237800849078-rmntmr1b2tcu20kpid66q5dbh1vdt7aj.apps.googleusercontent.com&redirect_uri=http%3A%2F%2F127.0.0.1%3A5556%2Fauth%2Fgoogle%2Fcallback&response_type=code&scope=openid+email&state=8slZXeZhwKQofg%3D%3D
-Pushing signature to: gcr.io/user-vmtest2/demo:sha256-97fc222cee7991b5b061d4d4afdb5f3428fcb0c9054e1690313786befa1e4e36.sig
-```
-
-Keyless verifying:
-
-```shell
-$ COSIGN_EXPERIMENTAL=1 cosign verify gcr.io/user-vmtest2/demo
-The following checks were performed on all of these signatures:
-  - The cosign claims were validated
-  - The claims were present in the transparency log
-  - The signatures were integrated into the transparency log when the certificate was valid
-  - Any certificates were verified against the Fulcio roots.
-Certificate subject:  user@google.com
-{"Critical":{"Identity":{"docker-reference":""},"Image":{"Docker-manifest-digest":"sha256:97fc222cee7991b5b061d4d4afdb5f3428fcb0c9054e1690313786befa1e4e36"},"Type":"cosign container image signature"},"Optional":null}
-```
+Keyless signing uses ephemeral keys and certificates, which are signed automatically by the Fulcio root CA. Signatures are stored in the Rekor transparency log, which automatically provides an attestation as to when the signature was created.
```

Comment:
```suggestion
Keyless signing associates identities, rather than keys, with an artifact signature. Fulcio issues short-lived certificates binding an ephemeral key to an OpenID Connect identity. Signing events are logged in Rekor, a signature transparency log, providing an auditable record of when a signature was created.
```

Reply:
Rephrasing for clarity